wasm/src/account/private_key.rs

```diff
@@ -61,6 +61,16 @@ impl PrivateKey {
         let address = Address::try_from(self.private_key).unwrap();
         address.to_string()
     }
+
+    #[wasm_bindgen]
+    pub fn decrypt(&self, ciphertext: &str) -> Result<String, String> {
```

Comment:
If in the near future this function receives a `ViewKey` instead of a `PrivateKey` we could move this function to another module.

Reply:
I agree on this one, but I prefer to wait just to understand how this is going to be used since the `ViewKey` struct is  not being used